File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TurnTest.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
import com.qualcomm.robotcore.eventloop.opmode.Disabled;
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;

@Autonomous
@Disabled
public class TurnTest extends LinearOpMode {

    @Override
    public void runOpMode() throws InterruptedException {
        TeamHardwareMap hm = new TeamHardwareMap(this);

        waitForStart();

        hm.turnDegrees(360);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/tests/TurnTest.kt
package org.firstinspires.ftc.teamcode.tests

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import com.qualcomm.robotcore.eventloop.opmode.Disabled
import org.firstinspires.ftc.teamcode.autos.AutoBase

@Autonomous
@Disabled
class TurnTest : AutoBase() {
    override fun mainOpMode() {
        hw.waitForImuCalibration(false)

        hw.turnToHeading(90.0)
    }
}